openreview/conference/templates/recruitResponseWebfield.js

```diff
@@ -13,19 +14,19 @@ function main() {
   render();
 }
 
-
 function render() {
   var $response = $('#notes');
   $response.empty();
 
   if (args.response) {
     var accepted = (args.response === 'Yes');
-    var message = accepted ? 'Thank you for accepting the invitation!' : 'You have declined the invitation.';
-
-    $response.append('<div class="panel"><div class="row"><strong>' + message + '</strong></div></div>');
+    var declined = (args.response === 'No');
 
     if (accepted) {
       // Display response text
+      var message = 'Thank you for accepting this invitation from ' + HEADER.title;
+      $response.append('<div class="panel"><div class="row"><strong>' + message + '</strong></div></div>');
```

Comment:
This isn't that important, but if you're making any other change, I would prefer if this `strong` was changed to an `h3`. It looks a little funny in the screenshot.

Reply:
Done. I also applied the same style as other h3 tags on the page. Take a look.

![Screen Shot 2019-11-21 at 2 29 48 PM (2)](https://user-images.githubusercontent.com/13500158/69370197-78721d80-0c6b-11ea-8ba8-03eabf5eea87.png)

Reply:
Nice! Thanks!

I know it's the text that Thomas sent us, but it sounds a little off. Can we just remove the first sentence "It is unfortunate that you decline"?